tools/hubl/internal/remote.go

```diff
@@ -65,13 +66,16 @@ func RemoteCommand(config *config.Config, configDir string) ([]*cobra.Command, e
 			return nil, err
 		}
 
+		clientCtx := client.Context{}.
```

Comment:
When we have full tx support, we may need more things, but for now this is fine 👍🏾